snippets/msbuild-project.json

```diff
@@ -9,8 +9,153 @@
 		],
 		"description": "Load Metapackage for ASP.NET Core"
 	},
+	"ASP.NET Core Full Packages": {
+		"prefix": "aspnetcore-metapackage-full",
```

Comment:
Just a thought - perhaps you might want to make the prefix something like `aspnetcore-full` rather than including the term `metapackage` in the prefix? Might improve discoverability by helping consumers understand why they'd choose one over the other?

Reply:
Agreed. 👍